### PR TITLE
CARLOS: Added automatic vrx-docker evaluation script

### PR DESCRIPTION
**Pull request by MarshallRawson**
commit hashes no longer in the names of containers

    made vrx-trial containers dependent on commit hashes

    updated docs

    updated vrx and vrx-docker to have latest code

For demo, see proposed docs

### CODE REVIEW

File: docker/vrx_trial/vrx_trial_install [hunk 1]
```diff
@@ -19,9 +19,10 @@ MIL_REPO_DIR="$CATKIN_SOURCE_DIR/mil"
 mil_clone_repo()
 {
   sudo apt install -y git
-  git clone -b ${BRANCH} --single-branch ${GIT_URL} ${MIL_REPO_DIR}
+  git clone ${GIT_URL} ${MIL_REPO_DIR}
   cd ${MIL_REPO_DIR}
   git submodule update --init --recursive
```
Comment (kev-the-dev) [inline docker/vrx_trial/vrx_trial_install]: init submodule should happen after git reset so it fetches the correct submodules

File: docs/development/vrx.md [hunk 2]
```diff
@@ -33,26 +33,42 @@ The recommended workflow for submitting our code to vrx is:
...
+
+4. Run the vrx evaluation script witch will do every step of the evaluation specified by [the vrx docker repo](https://bitbucket.org/osrf/vrx-docker/src/default/) by running 
+
+	`./scripts/vrx_eval <git commit> my_nickname https://github.com/ME/mil.git me/vrx-repo`
+
+	* If your commit is on the ufmil git repo and you have access to the ufmil/vrx docker repo you can use a shorter version: *
+
+		`./scripts/vrx_eval <git commit> my_nickname`
+
+
```
Comment (kev-the-dev) [inline docs/development/vrx.md]: How do I know if this ran successfully? How do I pull logs / scores out? 

I don't see how someone could (using  the documentation provided) use this to validate code.

**Comment by MarshallRawson** [comment]
I think there is a bug in vrx.
run
`./scripts/vrx_eval <commit> <nickname>`
in another terminal run
`docker exec -it vrx-server-system bash`
`sudo gzclient`
you will see that the wamv urdf is broke and sand island isn't actually loaded.

File: NaviGator/mission_control/navigator_launch/launch/vrx.launch [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<!--
+    Launches the gazebo simulator and all the same master.launch used on the real platform.
```
Comment (kev-the-dev) [inline NaviGator/mission_control/navigator_launch/launch/vrx.launch]: this duplicate is still present...

**Comment by kev-the-dev** [comment]
Notify me if you get this working and want a re-review, otherwise this is stalled

**Pull request closed without merging**